docs/image/rundocs

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+pip install -r docs_requirements.txt
```

Comment:
Why is this executed here and in the [Dockerfile](https://github.com/deis/deis/pull/2887/files#diff-0f8ac35ad627718a859512ce1c59867cR10)? Removing this would also allow to remove this script and run `make server` directly.